Ghidra/Features/Decompiler/src/decompile/cpp/pcoderaw.hh

```diff
@@ -34,6 +34,8 @@ struct VarnodeData {
   AddrSpace *space;		///< The address space
   uintb offset;			///< The offset within the space
   uint4 size;                   ///< The number of bytes in the location
+  VarnodeData() = default;
+  VarnodeData(AddrSpace *space,uintb offset,uint4 size) : space(space), offset(offset), size(size) {}
```

Comment:
msvc refused to allow me to use `std::vector::emplace_back` without explicitly declaring the constructor. It really shouldn't have been necessary though.

The rule of 6 should be applied here but I didn't want to clutter it with a bunch of defaulted constructors.